pmoves/services/semantic-cache/cipher_layer.py

```
@@ -0,0 +1,90 @@
+                json={
+                    "content": query,
+                    "category": "context",
+                    "tags": ["semantic-cache"],
+                    "metadata": {
+                        "response_hash": resp_hash,
+                        "source": "semantic-cache",
+                    },
+                },
+                headers=self._headers(),
+            )
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)